src/middlewared/middlewared/plugins/afp.py

```diff
@@ -109,6 +110,7 @@ class Config:
         UnixPerm('umask', default='000'),
         List('hostsallow', items=[], default=[]),
         List('hostsdeny', items=[], default=[]),
+        Str('vuid', null=True, default=''),
```

Comment:
Do we need to expose vuid?

Reply:
Probably not, but we do it for SMB. It's nice to have the consistency. An advanced user may wish to convert a time machine share from AFP to SMB. In this case being able to transfer the VUID is useful.